express-post-json/index.js

```diff
@@ -0,0 +1,29 @@
+const express = require('express');
+const app = express();
+
+let nextId = 1;
+const grades = {};
+
+app.get('/api/grades', (req, res) => {
+  const array = [];
+  for (const key in grades) {
+    array.push(grades[key]);
+  }
+  res.json(array);
+});
+
+const JSON = express.json();
```

Comment:
You don't want to name this constant `JSON` because there is already a built in object with that same name and you don't want to overwrite that. Something like `jsonMiddleware` would be better.